vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,161 @@
+To secure the `getAllConferences` API:
 
-11. Go back into the `ConferenceResource.java` file and add this annotation to the `deleteConference` method:
+1. Go to `Users` → `View all users` → `admin` → `Role Mappings`
+2. Select `internal` for the `Client Roles` 
+3. Move `conference-user` from `Available Roles` to `Assigned Roles`
+4. Return to the MFE to confirm you see the full list of Conferences
 
+### Step 7: Restrict the ability to delete Conferences
+
+The `conference-user` role grants the admin user permission to delete Conferences. To restrict the delete method to the `conference-admin` role:
+
+1. Go to the `src/main/java/com/<ORG>/<NAME>.web.rest` directory
+2. Open `ConferenceResource.java` 
+3. Modify the `deleteConference` method by adding the following annotation
 ```
     @PreAuthorize("hasAuthority('conference-admin')")
     public ResponseEntity<Void> deleteConference(@PathVariable Long id) {
 ```
-Here we're restricting the delete method to only the `conference-admin` role.
 
-12. Restart the microservice. By default this will include rebuilding any changed source files.
-13. Once the microservice is available, go back to the MFE and try deleting one of the Conferences in the list. You should be able to attempt the delete in the UI but you'll get a 403 error in the browser console and an error like this in the service logs:
+To verify that a user without the `conference-admin` role is unable to call the delete API:
+
+1. Restart the microservice. By default this includes rebuilding any changed source files.
+2. Once the microservice is available, return to the MFE and try deleting one of the Conferences in the list 
+3. Verify that attempting to delete via the UI generates a `403 error` in the browser console and an error in the service logs similiar to the following:
 ```
 2021-03-22 15:56:16.205  WARN 3208 --- [  XNIO-1 task-3] o.z.problem.spring.common.AdviceTraits   : Forbidden: Access is denied
 ```
-That's exactly what we wanted! This demonstrates that a user without `conference-admin` is unable to call the delete API.
 
-Next, let's update the MFE so a user without the `conference-admin` authority cannot even see the delete button in the UI.
+### Step 8: Hide the delete button
 
-14. Edit the `ConferenceTableContainer.js` under `ui/widgets/conference/tableWidget/src/components`. Replace the onDelete logic with an additional check on the user's authorities.
+The MFE UI can be updated to hide the delete button from a user without the `conference-admin` authority. The key logic checks whether the `internal` client role `conference-admin` is mapped to the current user via the hasResourceRole call.
+
+1. Go to the `ui/widgets/conference/tableWidget/src/components` directory
+2. Open `ConferenceTableContainer.js` 
+3. Replace the `onDelete` logic with an additional user permissions
```

Comment:
add colon & delete 's' >> Replace the `onDelete` logic with an additional user permission: